packages/pwa/app/pages/cart/index.jsx

```diff
@@ -35,7 +35,7 @@ import {API_ERROR_MESSAGE} from '../../constants'
 import {REMOVE_CART_ITEM_CONFIRMATION_DIALOG_CONFIG} from './partials/cart-secondary-button-group'
 
 // Utilities
-import debounce from 'lodash.debounce'
+import debounce from 'lodash/debounce'
```

Comment:
We might want to implement this function ourselves since lodash is big in size to replace this one. [See comment](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/490/files#r842072387)

Reply:
Sounds good, 👍

Reply:
Also beyond scope, but I just want to note that transitive dependencies (relying on a dependency of a dependency) is a pattern that should be avoided.